prefect_docker/containers.py

```diff
@@ -0,0 +1,63 @@
+"""Integrations with Docker Containers."""
+
+from typing import Any, Container, Dict, List, Optional, Union
+
+from prefect import get_run_logger, task
+from prefect.utilities.asyncutils import run_sync_in_worker_thread
+
+from prefect_docker.host import DockerHost
+
+
+@task
+async def create_docker_container(
+    image: str,
+    command: Optional[Union[str, List[str]]] = None,
+    name: Optional[str] = None,
+    detach: Optional[bool] = None,
+    docker_host: Optional[DockerHost] = None,
```

Comment:
I think that `entrypoint` and `environment` we be used often enough that they can be raised to the level of named kwargs.
```suggestion
    image: str,
    command: Optional[Union[str, List[str]]] = None,
    entrypoint: Optional[Union[str, List[str]] = None,
    name: Optional[str] = None,
    detach: Optional[bool] = None,
    environment: Optional[Dict[str, str], List[str]] = None,
    docker_host: Optional[DockerHost] = None,
```